### PR TITLE
Movie suggestions generator

### PR DESCRIPTION
**Pull request by SophieSjoberg**
## What we have learned
We are definitely on, for us, uncharted territory. So problems will arise but so far do good. :)

### CODE REVIEW

**Review by ScottyyyB — APPROVED** [review]
Approve

**Review by ScottyyyB — CHANGES_REQUESTED** [review]
My bad...

File: app/controllers/landing_controller.rb [hunk 1]
```diff
@@ -1,4 +1,14 @@
 class LandingController < ApplicationController
   def index
+    # @movies = Tmdb { results:[] }
+  end
+
+  def search
+    genre = params[:genre].capitalize
+    binding.pry
```
Comment (zuzanna81) [inline app/controllers/landing_controller.rb]: Remove binding.pry

**Review by ScottyyyB — DISMISSED** [review]

**Comment by diraulo** [comment]
Why are we approving WIP PRs? 🤔

**Review by NegativeKarma — DISMISSED** [review]
All good!

File: Gemfile [hunk 2]
```diff
@@ -14,7 +14,9 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.2'
 gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
-
+gem 'awesome_print'
```
Comment (diraulo) [inline Gemfile]: Should probably move this to `:development, :test` group

File: app/controllers/landing_controller.rb [hunk 3]
```diff
@@ -1,4 +1,13 @@
 class LandingController < ApplicationController
   def index
+    # @movies = Tmdb { results:[] }
```
Comment (diraulo) [inline app/controllers/landing_controller.rb]: Let's remove this line

File: app/views/landing/index.html.erb [hunk 4]
```diff
@@ -3,5 +3,26 @@
...
+  <%= submit_tag 'search' %>
+<% end %>
+
+<% if @movies %>
+  <% if @movies.results.any? %>
+    <h2> Results </h2>
+      <ul>
+        <% @movies.results.take(5).each do |movie| %>
+        <li>
+          <img src="http://image.tmdb.org/t/p/w185<%=movie.poster_path %>" alt="" >
```
Comment (diraulo) [inline app/views/landing/index.html.erb:21]: move the image after the paragraph tag maybe. Not a blocker though, don't have to do it :)

**Review by diraulo — CHANGES_REQUESTED** [review]
minor changes and we're good to go. Remember to pull first before making those changes

File: Gemfile (hunk 2 above)
Comment (SophieSjoberg) [reply Gemfile]: okidoki!

File: app/controllers/landing_controller.rb (hunk 3 above)
Comment (SophieSjoberg) [reply app/controllers/landing_controller.rb]: Will do boss!

File: app/views/landing/index.html.erb (hunk 4 above)
Comment (SophieSjoberg) [reply app/views/landing/index.html.erb:21]: 👍

Comment (SophieSjoberg) [reply app/views/landing/index.html.erb:21]: Well didn't look good when I moved it so changed it back... :)

**Review by diraulo — APPROVED** [review]

**Pull request merged**